File: scala_core/src/org/jetbrains/plugins/scala/lang/refactoring/introduceVariable/ScalaIntroduceVariableSettings.scala
package org.jetbrains.plugins.scala.lang.refactoring.introduceVariable

import psi.types.ScType

/** 
* User: Alexander Podkhalyuzin
* Date: 01.07.2008
*/

trait ScalaIntroduceVariableSettings {
  def getEnteredName(): String

  def isReplaceAllOccurrences(): Boolean

  def isDeclareVariable(): Boolean

  def getSelectedType(): ScType
}

File: scala_core/src/org/jetbrains/plugins/scala/lang/refactoring/introduceVariable/ScalaIntroduceVariableSettings.java
package org.jetbrains.plugins.scala.lang.refactoring.introduceVariable;

import com.intellij.psi.PsiType;

/**
 * User: Alexander Podkhalyuzin
 * Date: 24.06.2008
 */
public interface ScalaIntroduceVariableSettings {
  public String getEnteredName();

  public boolean isReplaceAllOccurrences();

  public boolean isDeclareVariable();

  public PsiType getSelectedType();
}
